connector/src/main/java/org/geysermc/connector/utils/InventoryUtils.java

```diff
@@ -98,4 +102,19 @@ public static boolean canStack(ItemData item1, ItemData item2) {
             return false;
         return item1.equals(item2, false, true, true);
     }
+
+    /**
+     * Returns a barrier block with custom name and lore to explain why
+     * part of the inventory is unusable.
+     */
+    public static ItemData createUnusableSpaceBlock(String description) {
+        CompoundTagBuilder root = CompoundTagBuilder.builder();
+        CompoundTagBuilder display = CompoundTagBuilder.builder();
+
+        display.stringTag("Name", ChatColor.RESET + "Unusable inventory space");
+        display.listTag("Lore", StringTag.class, Collections.singletonList(new StringTag("", ChatColor.RESET + ChatColor.DARK_PURPLE + description)));
+
+        root.tag(display.build("display"));
+        return ItemData.of(-161, (short) 0, 1, root.buildRootTag());
```

Comment:
Barrier ID should be a constant in `BlockTranslator`?

Reply:
The barrier index seemed to be the id for bread so I hardcoded it. And that’s the item id so it would go in ItemTranslator

Reply:
Yeah, ItemTranslator, my mistake.

Reply:
Fixed the hardcoded ID and another bug in https://github.com/lukeeey/Geyser/pull/1